jboss-kie-smartrouter/tests/bats/jboss-kie-smartrouter-common.bash

```diff
@@ -0,0 +1,9 @@
+load ../../../tests/bats/common/logging
+
+function compare(){
+  if cmp $file1 $file2; then
```

Comment:
it will always return true, to pass parameters to a function you should use $1, $2 and so on...
plus, using diff is better than this approach.

Reply:
ok